Formula/libbitcoin.rb

```diff
@@ -0,0 +1,59 @@
+class Libbitcoin < Formula
+  desc "Bitcoin Cross-Platform C++ Development Toolkit"
+  homepage "https://libbitcoin.org/"
+  url "https://github.com/libbitcoin/libbitcoin/archive/v3.3.0.tar.gz"
+  sha256 "391913a73615afcb42c6a7c4736f23888cfc999a899fc38395ddcbd560251d94"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "boost"
+
+  resource "secp256k1" do
+    url "https://github.com/libbitcoin/secp256k1/archive/v0.1.0.13.tar.gz"
+    sha256 "9e48dbc88d0fb5646d40ea12df9375c577f0e77525e49833fb744d3c2a69e727"
+  end
+
+  def install
+    resource("secp256k1").stage do
+      system "./autogen.sh"
+      system "./configure", "--disable-dependency-tracking",
+                            "--disable-silent-rules",
+                            "--prefix=#{prefix}",
+                            "--enable-module-recovery"
+      system "make", "install"
+      # Normally pkgconfig metadata for a dependency is available
+      # /usr/local/lib/pkgconfig, but Homebrew doesn't install it until the
+      # whole formula is complete.
+      ENV["PKG_CONFIG_PATH"] = "#{lib}/pkgconfig"
+    end
+    system "./autogen.sh"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <bitcoin/bitcoin.hpp>
+      int main() {
+        const auto block = bc::chain::block::genesis_mainnet();
+        const auto& tx = block.transactions().front();
+        const auto& input = tx.inputs().front();
+        const auto script = input.script().to_data(false);
+        std::string message(script.begin() + sizeof(uint64_t), script.end());
+        bc::cout << message << std::endl;
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "-std=c++11",
+                    "test.cpp",
```

Comment:
should be formatted like this: https://github.com/Homebrew/homebrew-core/blob/master/Formula/folly.rb#L62-L63